### ISSUE TITLE
Log Differentiation - Safari WiFi bug vs Location Denied (1 SP)

### ISSUE THREAD

**Issue by Kourchenko** [Hack4Eugene/SpeedUpAmerica#170]
**Is your feature request related to a problem? Please describe.**
As a developer looking at logs, I want to see a differentiation between when a user denies permission and when the permission denied is caused by browser incompatibilities. As such, we should add logic to log when an error is caused by location permission asked, then user denied and when an error is caused by simple location permission denied - which means the user experience should be improved as they were never asked for permissions.

**Describe the solution you'd like**
Develop logic to check and log whether Geolocation API requested permissions and received a success message.

**Describe alternatives you've considered**
An alternative is simply allowing our Sentry logging system to notify us if a user is using Safari, without differentiating in the logs themselves.

**Additional context**
This Issue is a simple logic adjustment; no UI updates to help the user understand that location is needed, but is being blocked by their browser. Another ticket will be made to adjust the logic.

**Comment by JCoulter85** [comment]
It looks like this issue has been resolved in a different ticket. Good to close. 👍
